ufo/__init__.py

```diff
@@ -25,13 +25,17 @@
   app.config['SQLALCHEMY_DATABASE_URI'] = os.environ['DATABASE_URL']
   app.config['WHOOSH_BASE'] = os.environ['DATABASE_URL']
 
+RECAPTCHA_ENABLED_FOR_APP = False
+MAX_FAILED_LOGINS_BEFORE_RECAPTCHA = 10
 if 'RECAPTCHA_SITE_KEY' in os.environ and 'RECAPTCHA_SECRET_KEY' in os.environ:
   app.config['RECAPTCHA_SITE_KEY'] = os.environ['RECAPTCHA_SITE_KEY']
   app.config['RECAPTCHA_SECRET_KEY'] = os.environ['RECAPTCHA_SECRET_KEY']
+  RECAPTCHA_ENABLED_FOR_APP = True
 else:
   app.logger.error('No recaptcha site or secret key found. Please configure ' +
                    'RECAPTCHA_SITE_KEY and RECAPTCHA_SECRET_KEY in the ' +
                    'environment variables.')
+  RECAPTCHA_ENABLED_FOR_APP = False
```

Comment:
No, it should be true if the variables are found and false otherwise. I set it to an initial value so that it is always defined, but I can confirm that it does still work when set to true. Plus we have tests for this.